types/react-fa/index.d.ts

```diff
@@ -37,7 +37,6 @@ export const Icon: ComponentClass<IconProps>;
 
 export interface IconStackProps extends SizeOverrideHTMLProps<IconStack> {
     size?: IconSize;
-    children?: IconProps[];
```

Comment:
People will have to uninstall this type definition if we don't fix this. it is a breaking change coming in, we cannot type children until there is support to do it. support is coming in 2.3 but it is not considered stable yet (even though it is out in the wild).

the breaking `@types/react` change: https://github.com/DefinitelyTyped/DefinitelyTyped/commit/13b5f660baa232a454a2496b43ed51299aea4a5b#diff-96b72df8b13a8a590e4f160cbc51f40c